### ISSUE TITLE
make the ''list recent tabs'' pop up upwards

### ISSUE THREAD

**Issue by Janaue** [jscher2000/switch-to-previous-active-tab#8]
I have firefox ESR and the trick to create an addon bar or status bar again by  invoking some files from this work https://github.com/Aris-t2/CustomJSforFx,
load_js_script("./CustomJSforFx-master/scripts/addonbar.uc.js");
in userChrome.uc.js
I also but the tab bar at the bottom of the window with
@import "./CustomCSSforFx-master/classic/css/tabs/tabs_below_main_content_fx74.css"; /**/
in userChrome.ccs

It works great and I place your icon on the addon bar to see the list of all recent tabs, but it doesn't show up. If I place the icon at the top of the window, like near the URL bar, then I see the list.
I other addons on this addon bar, like https://addons.mozilla.org/en-US/firefox/addon/tab-session-manager/ and they pop upwards to see their lists.

Can you do the same with yours ?

**Comment by jscher2000** [comment]
Do mean from the right-click context menu or did you change what the button does?

In order to offer the option to either using the button to click between tabs or to open the list, the popup is triggered from code and is not directly connected to the button. Maybe that's the reason it is behaving differently from your other extensions??

As far as I know, I can only choose whether to use a drop-down panel or a popup window, I can't specify how the panel opens.

**Comment by Janaue** [comment]
yes I mean the right click on the icon, then I select ''list of recent tabs'' to get the pop-up. When the icon is at the top of the window, the pop up is displayed in full, but when the icon is at the bottom, there is only a thin layer of the top of the pop-up which appears.

I have stock options for this addon.  The same thing happens with other entries from the right-click, like ''options'' and ''reload all tabs options''

**Comment by Janaue** [comment]
I tried to change the behavior of the icon, by choosing to display the pop up directly, and it still doesn't appear (ie only a thin layer of what would be the top of the pop-up)

**Comment by Janaue** [comment]
I managed to get the pop-up for the ''options'' and ''list of recent tabs'' by clicking a lot on the icon, but even if the pop-up appears, after i close it, it struggles to be displayed properly afterwards.
